content/en/docs/instrumentation/java/springboot.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Spring Boot
+linkTitle: Spring Boot
+aliases:
+  - /docs/java/getting_started
+  - /docs/java/springboot
+  - /docs/instrumentation/java/springboot
+weight: 30
+description: Spring instrumentation for OpenTelemetry Java
+# prettier-ignore
+---
+
+You can instrument a Spring Boot application with the [OpenTelemetry Java agent](automatic/_index).
+
+Another option is to use the OpenTelemetry Spring Boot starter. The rest of this page document it.
+
+## Getting Started
+
+OpenTelemetry Spring Starter is a starter package that allow you to setup distributed tracing.
+
+Check out [opentelemetry-spring-boot-autoconfigure](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/README.md#features) for the full list of supported libraries and features.
```

Comment:
As someone who is not involved here, this is confusing, because the link you share brings me to something called "OpenTelemetry Spring Auto-Configuration", which then links to "OpenTelemetry Spring Starter".

Also, people might not know what a "Starter" and what a "Auto-Configuration" is. Do we have anything we can reference here first to clarify what they are going to get here (especially also compared to the agent)